gnosis/safe-react
# Use subtabs in sidebar

## What it solves
Resolves #1295 

## How this PR fixes it
This PR adds subtabs menu for ASSETS and SETTINGS, making easier to discover collectibles and making the view for the settings bigger

## How to test it
Just load a safe and navigate using the sidebar menu. Check for the expected behavior when clicking around

## Screenshots
Notification badge example for current PR
![image](https://user-images.githubusercontent.com/25051234/125272925-adcdf200-e30c-11eb-9a90-80fdcb409454.png)
